config/manifests/bases/awx-operator.clusterserviceversion.yaml

```
@@ -50,6 +50,10 @@ spec:
         path: ingress_annotations
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:text
+      - displayName: Route Annotations
+        path: route_annotations
+        x-descriptors:
+          - urn:alm:descriptor:com.tectonic.ui:text
```

@sharvarikhedkar I think we should change this to only show if `ingress_type: Route` is selected.

```suggestion
      - displayName: Route Annotations
        path: route_annotations
        x-descriptors:
          - 'urn:alm:descriptor:com.tectonic.ui:advanced'
          - 'urn:alm:descriptor:com.tectonic.ui:text'
          - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:ingress_type:Route'
```

---

updated.